### PR TITLE
Fix/3d max sp5

### PR DESCRIPTION
**Pull request by mycheckgithub**
Only ColladaMax project has configuration with SP5 for x64 (no more Win32)
(ColladaBaseUtil.vcproj and others projs have no more configuration changes)

#define SP5 has been changed by MAX_2014_SP5

There is no SDK specific to SP5. The tricks is to compile Max2014_SP5 with  "morpher_2015_x64.lib" and not "morpher_2014_x64.lib" and modify BuildFromNode declaration based on "MAX_2014_SP5_OR_NEWER "

### CODE REVIEW

**Comment by mycheckgithub** [comment]
ColladaMAX build FAILED because new jenkins jobs with max 2017 have been created.
So i cherry pick from upstream Khronos misc/add-max-2017

File: COLLADAMax/scripts/COLLADAMax.vcxproj [hunk 1]
```diff
@@ -193,6 +225,14 @@
       <Configuration>Release_Max2016</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release_Max2017|Win32">
```
Comment (RemiArnaud) [inline COLLADAMax/scripts/COLLADAMax.vcxproj]: This config does not exist (win32 2017) - not a problem for this PR, but I should look into this at later time

File: COLLADAMax/scripts/COLLADAMax.vcxproj [hunk 2]
```diff
@@ -1559,6 +1744,41 @@
...
     </PostBuildEvent>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug_Max2014_SP5|Win32'">
+    <PreBuildEvent>
+      <Command>call create_version_info_h.bat $(PLATFORM) $(CONFIGURATION)</Command>
+    </PreBuildEvent>
+    <ClCompile>
+      <Optimization>Disabled</Optimization>
+      <AdditionalIncludeDirectories>..\include;..\res;$(MAX_SDK_PATH2014)\include;..\..\COLLADAFramework\include;..\..\COLLADAFramework\include\Math;..\..\COLLADADomHelper\include;..\..\COLLADAStreamWriter\include;..\..\COLLADABaseUtils\include;..\..\GeneratedSaxParser\include;..\..\COLLADASaxFrameworkLoader\include;..\..\G3DWarehouseBrowser\include;$(BOOST_ROOT)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>WIN32;_DEBUG;_WINDOWS;_USRDLL;IWILLNOTUSEASSERTSOUTSIDETHISSCOPE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```
Comment (RemiArnaud) [inline COLLADAMax/scripts/COLLADAMax.vcxproj]: looks like its missing the MAX_2014_SP5 define to trigger the SP5 build?

**Comment by RemiArnaud** [comment]
Added debug and static builds to jenkins, launching build again

**Comment by RemiArnaud** [comment]
retest this please

**Comment by RemiArnaud** [comment]
test this please

**Comment by RemiArnaud** [comment]
ok to test

**Comment by RemiArnaud** [comment]
test this please

**Comment by RemiArnaud** [comment]
note - this succesfuly compiled 2014 SP5 for debug/release/static.
I have notified the bug reporters to test this exporter
I will now update the master build job to include those targets and merge this PR.

**Pull request merged**